src/backend/app/users/user_routes.py

```diff
@@ -262,3 +265,84 @@ async def reset_password(
         content={"detail": "Your password has been successfully reset!"},
         status_code=200,
     )
+
+
+@router.post("/regulator/", tags=["Auto Regulator Account Creation"])
+async def regulator_create(
+    db: Annotated[Connection, Depends(database.get_db)], data: dict
```

Comment:
we should use a Pydantic class to validate the token as Base64 instead of passing data as a plain dictionary.